sktime/tests/_config.py

```diff
@@ -37,6 +37,13 @@
     "SimpleRNNClassifier",
     "SimpleRNNRegressor",
     "EditDist",
```

Comment:
I notice, EditDist is not a DL estimator - do we want to remove it from the list or write a separate comment?